gtda/homology/simplicial.py

```diff
@@ -86,8 +87,8 @@ class VietorisRipsPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
 
     See also
     --------
-    SparseRipsPersistence, EuclideanCechPersistence, CubicalPersistence,
-    ConsistentRescaling
+    FlagserPersistence, SparseRipsPersistence, EuclideanCechPersistence, \
```

Comment:
Would you also think that simplicial persistence transformers should not be referred to in the "See also" section of `CubicalPersistence`?

Reply:
Yes, I think this was the case when we only had `CubicalPersistence` and `VietorisRipsPersistence`.

I agree, `CubicalPersistence` should not be in the "See also" section of simplicial transformers and simplicial transformers and vice-versa.